.github/workflows/tag.yml

```diff
@@ -0,0 +1,27 @@
+name: Tag
+on:
+  workflow_run:
+    workflows: [Build]
```

Comment:
> Build runs for PRs and stuff as well.

That's true. During testing, it just runs for the default branch so that we can avoid that situation

Reply:
But will that not release for every PR in this repo? Out builds run for all PRs as well as main.

Reply:
I forgot to reply, apologies. I didn't think so but my knowledge of GitHub actions is limited. Perhaps to be safe, we adjust the settings to be something like "only run the release workflow when we merge a PR to he main branch". I'm not sure if that's possible. I'll have to look into it

Reply:
I should have been more clear, and not try to be educational 😛 (I usually do questions like the above instead of stating things directly. Probably a force of habit after being a teacher for a while). If we do not specify a branch, it will in theory run for all branches. Just like you have specified type and workflows above, you can also specify `branches` 🙂 
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#limiting-your-workflow-to-run-based-on-branches